### PR TITLE
Support for `PseudospecCtrl` as well as `spectralWindow` and `uniform`

### PR DESCRIPTION
**Pull request by poulson**

### CODE REVIEW

File: src/julia/darray.jl [hunk 1]
```diff
@@ -128,6 +161,21 @@ function foxLi{T<:BlasComplex}(::Type{T}, n::Integer, ω::Real)
 end
 foxLi(n::Integer, ω::Real) = foxLi(Complex128, n, ω)
 
+function uniform{T<:BlasFloat}(::Type{T}, m::Integer, n::Integer)
```
Comment (andreasnoack) [inline src/julia/darray.jl]: I don't think we need to call Elemental to generate uniformly distributed variates. Something like `uniform{T}(::Type{T}, (m,n)) = scale!(drand(T, (m,n)) - T(0.5), 2)` should be fine, I think.

**Comment by andreasnoack** [comment]
@poulson Except for my small comment, is this then ready to get merged?

File: src/julia/darray.jl (hunk 1 above)
Comment (poulson) [inline src/julia/darray.jl]: What guarantees does `drand` make about the independence of each process's random number stream? In C this leads to catastrophic problems.

Comment (andreasnoack) [inline src/julia/darray.jl]: You have a point here, but it is really something that should be handled within Julia's existing framework for random number and DArrays. I think we a doing okay, i.e. using different seed for our Mersenne-Twister on each worker.

What are you doing in Elemental to ensure independece?

Comment (poulson) [inline src/julia/darray.jl]: I am essentially doing the exact same thing (see `src/core/global.cpp`), so I am A-okay with using `drand` instead. Would you mind submitting the revision?

Comment (andreasnoack) [inline src/julia/darray.jl]: I'll make that change. Is the rest good to go?

Comment (poulson) [inline src/julia/darray.jl]: Yes, everything was working for me.

**Comment by andreasnoack** [comment]
@poulson Thanks for these. I ended up removing the `uniform(DArray)` method because the functionality is already almost there with `drand` and I'd like to keep methods defined for `DArray` as consistent as possible. The methods defined for subtypes of `ElementalMatrix` follows your API much closer.

**Comment by poulson** [comment]
Great! Thanks.

**Pull request merged**